crates/uv/tests/pip_sync.rs

```diff
@@ -1801,7 +1803,7 @@ fn install_url_built_dist_cached() -> Result<()> {
     ----- stdout -----
 
     ----- stderr -----
-    Removed 2 files for tqdm ([SIZE])
+    Removed 3 files for tqdm ([SIZE])
```

Comment:
I don't know why this changed. Will look into it.

Reply:
We have an additional `.msgpack` file from the cache, because we fetch the `Requires-Python` for the URL.